clients/web/test/testEnv.pug

```diff
@@ -14,7 +14,7 @@ html(lang="en")
     each js in jsFiles
       script(src=js)
     
-    script(src="../../../test/lib/promise.min.js")
```

Comment:
No, I use Core.min.js to have a super polyfill for Phantom.js now because Vega uses something about Symbol. Did I forget to remove it from some place?